ui/src/components/setlist/adminView/SetlistViewContainer.tsx

```diff
@@ -17,14 +19,35 @@ const SetlistButton = styled(Button)(({ theme }) => ({
   padding: '8px 24px',
 }));
 
+const formatDate = (dateString: string | number | Date) => {
+  const date = new Date(dateString);
+  const year = date.getFullYear();
+  const month = (date.getMonth() + 1).toString().padStart(2, '0');
+  const day = date.getDate().toString().padStart(2, '0');
+
+  return `${year}-${month}-${day}`;
+};
+
```

Comment:
[minor]: dis is a very useful function, I think u can refactor this to the helpers folder and  reuse this for the dates in the folder drawer instead of crafting it manually?